specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json

```diff
@@ -282,7 +302,12 @@
             "description": "Accepted"
           }
         },
-        "x-ms-long-running-operation": true
+        "x-ms-long-running-operation": true,
```

Comment:
Add x-ms-long-running-operation-options

Reply:
this doesn't seem like a strict requirement (should have, not must have). We have been adding several x-ms-long-running-operation in several APIs without options without this being a concern (is this new?). In our case, the default value used (azure-async-operation) is exactly what we want.

If possible, I would strongly prefer to take an item to make this change in the future as it will require some thought in our swagger generation code.